src/content/en/updates/2017/04/native-mac-os-notifications.md

```diff
@@ -0,0 +1,141 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: As of Chrome version 59, notifications will be displayed native notifications on MacOS.
+
+{# wf_updated_on: 2017-04-26 #}
+{# wf_published_on: 2017-04-26 #}
+{# wf_tags: chrome59,notifications,push #}
+{# wf_featured_image: /web/updates/images/generic/notifications.png #}
+{# wf_featured_snippet: As of Chrome version 59, notifications will be displayed native notifications on MacOS. #}
+
+{% include "web/tools/chrome-devtools/_shared/styles.html" %}
+
+# Moving to the the Native Notification System on MacOS {: .page-title }
+
+Starting in Chrome 59, notifications sent via the [Notifications
+API](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API) or the
+[chrome.notifications extensions
+API](https://developer.chrome.com/extensions/notifications) will be shown
+directly by the MacOS native notification system instead of Chrome's own system.
+
+This change makes Chrome on MacOS feel much better integrated into the platform
+and fixes a number of long standing bugs, such as Chrome not respecting the
+system Do Not Disturb setting.
+
+Below we'll look at the differences this change introduces to the existing
+API's.
+
+## Notification Center
+
+One of the benefits (and feature requests from users) of this change is that
```

Comment:
(and feature requests from users) doesn't seem to fit logically or grammatically in the sentence. Can you take another shot at this.